lib/reline/windows.rb

```diff
@@ -92,6 +92,8 @@ def call(*args)
   @@ReadConsoleInput = Win32API.new('kernel32', 'ReadConsoleInput', ['L', 'P', 'L', 'P'], 'L')
   @@GetFileType = Win32API.new('kernel32', 'GetFileType', ['L'], 'L')
   @@GetFileInformationByHandleEx = Win32API.new('kernel32', 'GetFileInformationByHandleEx', ['L', 'I', 'P', 'L'], 'I')
+  @@hConsoleOutputHandle = @@GetStdHandle.call(STD_OUTPUT_HANDLE)
```

Comment:
`@@GetStdHandle.call(STD_OUTPUT_HANDLE)` has already been called in the line above, and `@@hConsoleOutputHandle` is not used. This line should be removed.